target/rv64gc/testlist.yaml

```diff
@@ -53,6 +53,18 @@
     +num_of_sub_program=5
   rtl_test: core_base_test
 
+- test: riscv_invalid_csr_test
```

Comment:
missing iteration count?

Reply:
You're right, I've added the base iteration count of 2.